pynicotine/gtkgui/widgets/textentry.py

```diff
@@ -179,18 +174,17 @@ def get_completions(part, completion_list):
 
     def remove_completion(self, item):
 
-        if not config.sections["words"]["tab"]:
-            return
+        if config.sections["words"]["tab"] and item in self.completion_list:
+            self.completion_list.remove(item)
 
-        if item not in self.completion_list:
+        if not config.sections["words"]["dropdown"]:
             return
 
-        self.completion_list.remove(item)
+        iterator = self.completion_iters.get(item)
 
-        if not config.sections["words"]["dropdown"]:
+        if iterator is None:
```

Comment:
Ditto

Reply:
Without the check it results in `TypeError: Argument 1 does not allow None as a value`